### ISSUE TITLE
Reformatting of the {-# OVERLAPPING #-} pragma

### ISSUE THREAD

**Issue by ptek** [commercialhaskell/hindent#386]
I have a similar issue to #385. Probably even the same cause.

Hindent changes this code:

```haskell
instance {-# OVERLAPPING #-} Arbitrary (Set Int) where
    arbitrary = ...
```

to 

```haskell
instance Arbitrary (Set Int) {-# OVERLAPPING #-}
                             where
    arbitrary = ...
```

So when compiling, GHC rightfully complains that it can’t parse `{-# OVERLAPPING`

Would you mind reopening the issue?

**Comment by sighingnow** [comment]
It's a HSE's bug, see haskell-suite/haskell-src-exts#354.

**Comment by NorfairKing** [comment]
+1

**Comment by sighingnow** [comment]
This issue has already been fixed in 432c54403ff92.
